cloud/jenkins/version_service_docker_build.groovy

```diff
@@ -0,0 +1,142 @@
+void checkImageForDocker(){
+     withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
+        sh '''
+            IMAGE=\$(cat IMG)
+            TrityHightLog="$WORKSPACE/trivy-hight-\$(echo \$IMAGE | tr -d '/:').log"
+            TrityCriticaltLog="$WORKSPACE/trivy-critical-\$(echo \$IMAGE | tr -d '/:').log"
+
+            sg docker -c "
+                docker login -u '${USER}' -p '${PASS}'
+                /usr/local/bin/trivy -q --cache-dir /mnt/jenkins/trivy-${JOB_NAME}/ image -o \$TrityHightLog --timeout 10m0s --ignore-unfixed --exit-code 0 --severity HIGH \$IMAGE
+                /usr/local/bin/trivy -q --cache-dir /mnt/jenkins/trivy-${JOB_NAME}/ image -o \$TrityCriticaltLog --timeout 10m0s --ignore-unfixed --exit-code 0 --severity CRITICAL \$IMAGE
+            "
+
+            if [ ! -s \$TrityHightLog ]; then
+                rm -rf \$TrityHightLog
+            fi
+
+            if [ ! -s \$TrityCriticaltLog ]; then
+                rm -rf \$TrityCriticaltLog
+            fi
+        '''
```

Comment:
please try to use new approach for trivy (you can check all our docker build jobs) and you will get  really good report

Reply:
Fixed. Please re-check.